chain/beacon/node.go

```diff
@@ -457,6 +459,7 @@ func (h *Handler) broadcastNextPartial(ctx context.Context, current roundInfo, u
 			h.l.Debugw("sending partial", "round", round, "to", i.Address())
 			err := h.client.PartialBeacon(ctx, &i, packet)
 			if err != nil {
+				metrics.ErrorSendingPartial(beaconID, round, i.Address())
```

Comment:
What if this time `err == nil` but last round it wasn't? Do we ever reset this?

Reply:
we don't reset it, we just measure the difference over time to work out when a node is not receiving partials and thus "down"

Reply:
Ah hmm I see what you're saying - let me double check if my understanding of this metric is right

Reply:
I still think counter is easier to use here: essentially every time unit we do something like `count(diff(count) >0 group by address)`. Given we won't be monitoring this at less than `period` it seems prudent and avoids us having to manage inc/dec of a gauge

Reply:
An alternative is to use a gauge and inc/dec on each partial and group by address possibly?